automatic/mkdocs/tools/ChocolateyInstall.ps1

```diff
@@ -1,10 +1,34 @@
 ﻿Update-SessionEnvironment
-
+$allowed_python_versions = @('3.9', '3.8', '3.7', '3.6', '3.5') # sync with nuspec
 $version = '1.1.2'
 
 $proxy = Get-EffectiveProxy
 if ($proxy) {
   Write-Host "Setting CLI proxy: $proxy"
   $env:http_proxy = $env:https_proxy = $proxy
 }
-python -m pip install mkdocs==$version
+
+function FindPython {
+  param(
+    $allowed_python_versions
```

Comment:
If you're going to use an Advanced Function, can you use `[Parameter()]`, and include the variable type?